static/src/javascripts/lib/geolocation.js

```diff
@@ -408,6 +408,35 @@ const extendedCurrencySymbol = {
 const getLocalCurrencySymbol = (): string =>
     extendedCurrencySymbol[countryCodeToCountryGroupId(getSync())] || '£';
 
+const countryNames = {
+    GB: 'The United Kingdom',
```

Comment:
The definitive ('The') should be lower case, please confirm @ionamckendrick - ditto for the US, NL, AE.
So it should read: `our readers in the United Kingdom` rather than `our readers in The United Kingdom`

Reply:
Sorry I missed this. Good spot @jlieb10 !